File: src/053. Combinatoric selections/053.rb
factorials = [1]
1.upto(100) do |n| 
  factorials[n] = factorials[n - 1] * n
end

count = 0
1.upto(100) do |n|
  1.upto(n) do |r|
    count += 1 if factorials[n] / factorials[r] / factorials[n - r] > 1_000_000
  end
end

puts count

File: src/053. Combinatoric selections/053.py
factorials = [1]
for n in range(1, 101):
    factorials.append(factorials[n - 1] * n)

count = 0
for n in range(1, 101):
    for r in range(1, n + 1):
        if factorials[n] / factorials[r] / factorials[n - r] > 1000000:
            count += 1

print(count)
